config/rubocop.yml

```diff
@@ -204,9 +204,15 @@ Style/TrailingBlankLines:
 Style/TrailingCommaInArguments:
   Enabled: true
 
-Style/TrailingWhitespace:
+Layout/TrailingWhitespace:
   Enabled: true
 
 Lint/EndAlignment:
-  AlignWith: variable
+  EnforcedStyleAlignWith: variable
   Enabled: true
+
+Style/SingleLineBlockParams:
+  Enabled: true
+
+Style/SymbolArray:
+  Enabled: false
```

Comment:
So first 2 got renamed, you got that from the changelog?
What about this SingleLineBlockParams and SymbolArray?
A dependency updating commit shouldn't introduce new rules, although that's the only thing stated in the changelog. So what are these? :)